requirements.txt

```diff
@@ -8,13 +8,16 @@ absl-py==0.9.0            # via nanoemoji (setup.py)
 appdirs==1.4.4            # via fs
 attrs==19.3.0             # via ufolib2
 booleanoperations==0.9.0  # via ufo2ft
+cairosvg==2.4.2
```

Comment:
cairosvg and pillow should be added to the setup.py install_requires and then the requirements.txt regenerated via `pip-compile setup.py`, see also https://github.com/googlefonts/nanoemoji/pull/90/commits/64841b49003841eaba1d29de52711441730cfd4c#r462409939

Reply:
Done and wrote https://googlefonts.github.io/python to try to learn how some of the parts fit together :)